test/v3/USTBv3MainnetForkTests.t.sol

```diff
@@ -75,35 +76,4 @@ contract USTBv3MainnetForkTest is Test {
         // ensure balance state is the same
         assertEq(404619420184, token.balanceOf(capturedMainnetAddress));
     }
-
-    function testUpgradeAndChangeOwner() public {
```

Comment:
wat happened to this one?

Reply:
there's no ownership change from v2 to v3